src/Microsoft.Health.Operations.Functions/Management/OrchestrationInstanceMetadata.cs

```diff
@@ -0,0 +1,35 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using EnsureThat;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask;
+
+namespace Microsoft.Health.Operations.Functions.Management;
+
+public sealed class OrchestrationInstanceMetadata
```

Comment:
This type mirrors the new OrchestrationMetadata class in DTFx's isolated worker API